runtime/README.md

```diff
@@ -1,53 +1,55 @@
 # Caliptra Runtime Firmware
 
-## Runtime Firmware Environment
+This specification describes the Caliptra Runtime Firmware. 
 
-### Boot & Initialization
+## Runtime Firmware environment
 
-The Runtime Firmware main function SHALL perform the following on Cold Boot reset:
+This section provides an overview of the Runtime Firmware environment.
 
-* Initialize DPE (see below for details)
+### Boot and initialization
+
+The Runtime Firmware main function SHALL perform the following on cold boot reset:
+
+* Initialize the [DICE Protection Environment (DPE)](#dice-protection-environment-dpe)
 * Initialize any SRAM structures used by Runtime Firmware
 
-For behavior during other types of reset, see "Runtime Firmware Updates".
+For behavior during other types of reset, see [Runtime firmware updates](#runtime-firmware-updates).
 
-If `mailbox_flow_done` is not set during a "warm" "update" boot, it is assumed that Caliptra
-was reset while runtime firmware was executing an operation. If Runtime firmware detects
-this case, it will call `DISABLE_ATTESTATION`, since the internal state of Caliptra may
+If `mailbox_flow_done` is not set during a warm update boot, it is assumed that Caliptra
+was reset while Runtime Firmware was executing an operation. If Runtime Firmware detects
+this case, it calls `DISABLE_ATTESTATION` because the internal state of Caliptra may
 be corrupted.
 
-### Main Loop
+### Main loop
 
-After booting, Caliptra Runtime firmware is responsible for the following:
+After booting, Caliptra Runtime Firmware is responsible for the following.
 
-* Wait for mailbox interrupts
-* On mailbox interrupt
-    * Unset `mailbox_flow_done`
-    * Write lock mailbox and set busy register
-    * Read command from mailbox
-    * Execute command
-    * Write response to mailbox and set necessary status register(s)
-    * Set `mailbox_flow_done`
-    * Sleep until next interrupt
-* On panic
-    * Save diagnostic information
+* Wait for mailbox interrupts. On mailbox interrupt, Runtime Firmware:
+    * Unsets `mailbox_flow_done`
+    * Writes lock mailbox and set busy register
+    * Reads command from mailbox
+    * Executes command
+    * Writes response to mailbox and set necessary status registers
+    * Sets `mailbox_flow_done`
```

Comment:
I think we can remove all references to `mailbox_flow_done` and `mailbox_data_avail`. These are hardware signals and not controlled by firmware. Also I think the hardware spec sufficiently describes how the hardware mailbox procol works.